turbopack/crates/turbopack-ecmascript/src/tree_shake/graph.rs

```diff
@@ -408,7 +408,7 @@ impl DepGraph {
 
                     for other_item in other_group {
                         if let ItemId::Group(ItemIdGroupKind::Export(export, _)) = other_item {
-                            if !export.0.as_str().starts_with("$$RSC_SERVER_") {
+                            if !export.0.as_str().starts_with("$$ACTION_") {
```

Comment:
This is the new prefix for both action and cache, x-ref: https://github.com/vercel/next.js/pull/70257

We need to use the `$RSC_SERVER_` prefix

Reply:
Oops it's a rebase mistake.